staff/web/vhost/redirect_check.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+import argparse
+import socket
+
+import requests as r
+from ocflib.vhost.web import get_vhosts
+
+out = 'redirect.log'
+errors = 'errors_redir.log'
+special_strings = ['asuc', 'ocf']
+
+
+def striphttp(name):
+    return name.split('/')[2]
+
+
+def is_special(url_string):
+    return any([special_string in url_string for special_string in
+                special_strings])
+
+
+def check_vhosting():
+    with open(out, 'w') as log, open(errors, 'w') as error_file:
+        vhosts = get_vhosts()
+        vhost_urls = []
+        for vhost_url in vhosts.keys():
+            if any(is_special(url) for url in {vhost_url} | set(vhosts[vhost_url]['aliases'])):
```

Comment:
FYI you can also write `{vhost_url, *vhosts[vhost_url]['aliases']}` since Python 3.5